src/mmw/js/src/core/templates/header.ejs

```diff
@@ -13,7 +13,7 @@
                     </a>
                     <ul class="dropdown-menu">
                         <li><a href="#">Profile</a></li>
-                        <li><a href="accounts/logout">Logout</a></li>
+                        <li><a href="javascript:void(0);" class="user-logout">Logout</a></li>
```

Comment:
IMO this should always link to the logout page so someone could right click on it and select "Copy URL" or maybe open it in a new tab. It's pretty easy to disable client-side if need be.